check-pulp-msgs.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+#
+# Developer ..........: Waldirio M Pinheiro <waldirio@redhat.com>
+# Created at .........: 05/01/2021 
+# Purpose ............: Check the qpid messages coming from/to Pulp.
+# Tested Sat Version .: 6.6, 6.7, 6.8, 6.9
+#
+
+# Below can be "Y" or "N"
+LOG_OUTPUT="N"
+
+SRV_FQDN=$(hostname)
+CERT="/etc/pki/pulp/qpid/client.crt"
+LOG="/var/log/pulp_queue.log"
+
+
+check_with_log()
+{
+while :
+do
+  date	| tee -a $LOG
+  qpid-stat --ssl-certificate=$CERT -b amqps://localhost:5671 -q | grep -E "(resource_manager.*$SRV_FQDN.*|^  queu|^  =|resource_worker.*$SRV_FQDN.*|resource_manager )" | tee -a $LOG
```

Comment:
You can consider making this a separate function:
```bash
qpid_stat() {
  qpid-stat --ssl-certificate=$CERT -b amqps://localhost:5671 -q | grep -E "(resource_manager.*$SRV_FQDN.*|^  queu|^  =|resource_worker.*$SRV_FQDN.*|resource_manager )"
}
```

Then in `check_with_log` you can run `qpid_stat | tee -a $LOG` and in without.

Another option is to write a log function:
```bash
#!/bin/bash

LOG=/tmp/log

write_log() {
	if [[ $LOG_OUTPUT == "Y" ]] ; then
		tee -a $LOG
	else
		cat
	fi
}

echo date | write_log
```

This then works:

```console
$ ./test.sh 
Wed  5 May 09:33:39 CEST 2021
$ LOG_OUTPUT=Y ./test.sh 
Wed  5 May 09:33:46 CEST 2021
$ cat /tmp/log 
Wed  5 May 09:33:46 CEST 2021
```

I think the latter is the cleanest result since you'll end up with just `check()` and `write_log()`. You may be able to find a better name than `write_log`, but it's about the principle :)

Reply:
Hello @ehokl, 
Thanks for the heads up and yea, I agree with you.

Changes applied and also I added something new that I believe to be interesting.

Thank you again.
Waldirio